src/components/Breadcrump.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "../styles/Breadcrumps.scss";
+
+function Breadcrumps({ page, variant, children }) {
+  if (page === "currencies-list") {
+    return variant === "details" ? (
+      <h3>
+        <Link to="/details" className="breadcrumps breadcrumps--unused">
+          {children}
+        </Link>
+      </h3>
+    ) : (
+      <h3>
+        <Link to="/" className="breadcrumps">
+          {children}
+        </Link>
+      </h3>
+    );
+  }
+  return variant === "details" ? (
+    <h3>
+      <Link to="/details" className="breadcrumps">
+        {children}
+      </Link>
+    </h3>
+  ) : (
+    <h3>
+      <Link to="/" className="breadcrumps breadcrumps--unused">
+        {children}
+      </Link>
+    </h3>
+  );
+}
```

Comment:
All of these conditions are redundant because in the code you still provide the values manually
```suggestion
function Breadcrumps({ linkTo, active, children }) {
  return (
    <h3>
      <Link
        to={linkTo}
        className={`breadcrumps ${active ? "" : "breadcrumps--unused"}`}
      >
        {children}
      </Link>
    </h3>
  );
}
```

it could be used like so:
```js
<Breadcrumps linkTo="/">Currencies list</Breadcrumps>/
<Breadcrumps linkTo="/details" active>
  Details
</Breadcrumps>
```

Reply:
Also as discussed on messenger, probably `--active` would make more sense than `--unused`

Reply:
changed